### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by DenysLiubchenko**
## Task
[3912](https://github.com/ita-social-projects/GreenCity/issues/3912)
## Changes
Add unit tests for ChatController and AzureFileServiceImpl

### CODE REVIEW

**Review by HelenSotnik — APPROVED** [review]

**Review by KizerovDmitriy — APPROVED** [review]

**Review by ABbondar — APPROVED** [review]

File: chat/src/test/java/greencity/controller/ChatControllerTest.java
Comment (VinuVicho) [inline chat/src/test/java/greencity/controller/ChatControllerTest.java:1]: We dont use star (*) imports

File: chat/src/test/java/greencity/controller/ChatControllerTest.java [hunk 1]
```diff
@@ -20,22 +20,16 @@
...
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
 
-import java.security.Principal;
-import java.util.ArrayList;
-import java.util.List;
-
 import static org.mockito.Mockito.*;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
```
Comment (VinuVicho) [inline chat/src/test/java/greencity/controller/ChatControllerTest.java]: Again * import

File: chat/src/test/java/greencity/service/impl/AzureFileServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.PropertyResolver;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
```
Comment (VinuVicho) [inline chat/src/test/java/greencity/service/impl/AzureFileServiceImplTest.java:22]: I recommend to change settings of IDEA, so it wouldn't suggest you star imports, as we don't use them)

**Review by VinuVicho — COMMENTED** [review]
Remove star imports)

**Pull request merged**